projects/ngx-datatable/src/lib/types/drag-events.type.ts

```diff
@@ -0,0 +1,11 @@
+export type DragEventType = 'drag' | 'dragend' | 'dragenter' | 'dragleave' |
+'dragover' | 'dragstart' | 'drop';
```

Comment:
Content of the same type should be indented if wrapped on a new line:

```suggestion
  'dragover' | 'dragstart' | 'drop';
```